cms/djangoapps/modulestore_migrator/rest_api/v1/views.py

```diff
@@ -117,28 +82,94 @@ def get_queryset(self):
     @apidocs.schema(
         body=ModulestoreMigrationSerializer,
         responses={
-            201: StatusWithModulestoreMigrationSerializer,
-            401: "The requester is not authenticated.",
+            201: StatusWithModulestoreMigrationsSerializer,
+            **_error_responses,
         },
-        summary="Start a modulestore to content library migration",
-        description=(
-            "Create a migration task to import course or legacy library content into "
-            "a content library.\n\n"
-            "**Request example**:\n\n"
-            "```json\n"
-            "{\n"
-            '  "source": "course-v1:edX+DemoX+2014_T1",\n'
-            '  "target": "library-v1:org1+lib_1",\n'
-            '  "composition_level": "unit",\n'
-            '  "repeat_handling_strategy": "update",\n'
-            '  "preserve_url_slugs": true\n'
-            "}\n"
-            "```"
-        ),
     )
     def create(self, request, *args, **kwargs):
         """
-        Handle the migration task creation.
+        Transfer content from course or legacy library into a content library.
+
+        This begins a migration task to copy content from a ModuleStore-based **source** context
+        to Learning Core-based **target** context. The valid **source** contexts are:
+
+        * A course.
+        * A legacy content library.
+
+        The valid **target** contexts are:
+
+        * A content library.
+        * A collection within a content library.
+
+        Other options:
+
+        * The **composition_level** (*component*, *unit*, *subsection*, *section*) indicates the highest level of
+          hierarchy to be transferred. Default is *component*. To maximally preserve the source structure,
+          specify *section*.
+        * The **repeat_handling_strategy** specifies how the system should handle source items which have
+          previously been migrated to the target. Specify *skip* to prefer the existing target item, specify
+          *update* to update the existing target item with the latest source content, or specify *fork* to create
+          a new target item with the source content. Default is *skip*.
+        * Specify **preserve_url_slugs** as *true* in order to use the source-provided block IDs
+          (a.k.a. "URL slugs", "url_names").  Otherwise, the system will use each source item's title
+          to auto-generate an ID in the target context.
+        * Specify **forward_source_to_target** as *true* in order to establish a mapping from the source items to the target
+          items. This is useful, for example, if you want to migrate a legacy library into content library while
+          preserving the integrity of course references to the original legacy library. Otherwise, no source-to-target
+          mapping will be established.
```

Comment:
Could we explain how this mapping is stored / represented / used? i.e. does this modify the source or create an entry in a separate mapping table?